src/utils/ulid.ts

```diff
@@ -45,34 +39,22 @@ function encodeTime (now: number, len: number): string {
   return str
 }
 
-/**
- * encodes random character
- * @param len
- * @returns {string}
- */
-function encodeRandom (len: number): string {
+function encodeRandom(len: number): string {
   let str = ''
   for (; len > 0; len--) {
     str = randomChar() + str
   }
   return str
 }
 
-/**
- * gets a a random charcter from generated pseudorandom number
- * @returns {string}
- */
-function randomChar (): string {
+function randomChar(): string {
   let rand = Math.floor(prng() * ENCODING_LEN)
   if (rand === ENCODING_LEN) {
     rand = ENCODING_LEN - 1
   }
   return ENCODING.charAt(rand)
 }
 
-/**
- * the factory to generate unique identifier based on time and current pseudorandom number
- */
```

Comment:
Looks good, just in case it wasn't intended there are comments like this one being removed.